internal/uvm/create_wcow.go

```diff
@@ -26,6 +27,22 @@ type OptionsWCOW struct {
 	*Options
 
 	LayerFolders []string // Set of folders for base layers and scratch. Ordered from top most read-only through base read-only layer, followed by scratch
+
+	// specifies if this UVM will be saved as a template in future. Setting this
```

Comment:
Rephrase these comments to start with the field name e.g. `IsTemplate specifies if this UVM...`.

This is typically how doc comments are written in Go.